modules/kogito-trusty-postgresql/module.yaml

```
@@ -0,0 +1,12 @@
+schema_version: 1
+name: org.kie.kogito.trusty.postgresql
+version: "2.0.0-snapshot"
+
+artifacts:
+  - name: trusty-service-postgresql-runner.jar
+    url: https://repository.jboss.org/nexus/service/local/repositories/snapshots/content/org/kie/kogito/trusty-service-postgresql/2.0.0-SNAPSHOT/trusty-service-postgresql-2.0.0-20211005.110152-1.jar
```

Please use same url as other modules

```suggestion
    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/trusty-service-postgresql/2.0.0-SNAPSHOT/trusty-service-postgresql-2.0.0-20211005.110152-1.jar
```